pkgs/development/python-modules/validphys2/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, nnpdf
+, reportengine
+}:
+
+buildPythonPackage rec {
+  pname = "validphys2";
+  version = "4.0";
+  format = "setuptools";
+
+  inherit (nnpdf) src;
+
+  prePatch = ''
+    cd validphys2
+  '';
+
+  postPatch = ''
+    substituteInPlace src/validphys/version.py \
+      --replace '= __give_git()' '= "${version}"'
+  '';
+
+  propagatedBuildInputs = [
+    nnpdf
```

Comment:
Only Python packages should be in `propagatedBuildInputs`.

Reply:
This now is a reference to `pythonPackages.nnpdf`